autobump-jgit/src/main/java/com/github/autobump/jgit/model/JGitGitClient.java

```diff
@@ -35,10 +34,10 @@ public Workspace clone(URI uri) {
     }
 
     @Override
-    public CommitResult commitToNewBranch(Workspace workspace, Bump bump) {
+    public CommitResult commitToNewBranch(Workspace workspace, String groupId, String versionNumber) {
```

Comment:
I see that in many places, a change was made to replace the Bump with 2 separate parameters, the groupId and the versionNumber.
I can't help but think that we might have missed an opportunity here to "embed" this in the Bump object. I'll explain better later today.